### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jk-fu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+   - Did you first encounter it recently, or has it always been there?
+
+   - If it is a UI issue, a screenshot or GIF will help tremendously. (For quick gifs see [http://www.cockos.com/licecap/](http://www.cockos.com/licecap/))
+   
+
+## About Package Updater
+
+Package Updater is application that helps developers safely (and _smartly_) update local packages in all their projects. Package Updater features help you easily update the dependencies so you can focus on coding more awesome things. 
+
+Read more on our website: [https://www.pkgup.io/](https://www.pkgup.io/)
```
Comment (igor-lemon) [inline README.md]: without `www`

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+---
+
+Say hello on Twitter (we're there as @pkgup). Twitter is where we will announce the latest news, features, and releases regarding Package Updater.
```
Comment (igor-lemon) [inline README.md]: Will you create a Twitter account?

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+> **Account Specific Queries:**
+> If you have any billing or account-specific queries, reach out to us by submitting a ticket on our [support page](https://www.pkgup.io/contact/).
+
+## Guidelines for reporting issues 
+
+Below is a short guideline to follow when reporting an issue. Following the guidline will help resolve issues in a timely manner. 
+
+1. If you are facing an account-related issue and you want to include personal information such as your username or collection names, you can reach out to us on our [support page](https://www.pkgup.io/contact/). 
+
+2. When reporting a new issue, it helps to answer the following questions:
```
Comment (igor-lemon) [inline README.md]: We can don't add it to readme, later I'll add the issue template

**Review by igor-lemon — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jk-fu) [reply README.md]: Twitter account created https://twitter.com/PkgUpdater

File: README.md (hunk 1 above)
Comment (jk-fu) [reply README.md]: Ok deleted

File: README.md (hunk 3 above)
Comment (jk-fu) [reply README.md]: Ok deleted

**Review by igor-lemon — APPROVED** [review]

**Pull request merged**